migrations/20171005171849_set_default_task_only_for_existing_form_stage.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class SetDefaultTaskOnlyForExistingFormStage extends AbstractMigration
+{
+
+    public function up()
+    {
+        $pdo = $this->getAdapter()->getConnection();
+        $rows = $this->fetchAll("SELECT task_is_internal_only from form_stages");
+        $insert = $pdo->prepare(
+            "UPDATE form_stages
+            SET task_is_internal_only = false"
+        );
+        foreach ($rows as $row) {
```

Comment:
@willdoran isn't this just going to iterate each row and do the same thing to all of them? I don't see a reason we need to run a foreach here since the update will have done that for all rows in the first run.

Reply:
hahahahaha...yes the foreach is irrelevant!

Reply:
Fixing